.travis.yml

```diff
@@ -1,9 +1,9 @@
 language: java
 jdk:
-   - oraclejdk8
+   - openjdk8
 script: mvn clean install --batch-mode
 matrix:
-  - jdk: oraclejdk8
+  - jdk: openjdk8
 branches:
   only:
     - master
```

Comment:
I completely agree with this change, but we should make it separately from this PR.